docs/apis/storm_compatibility.md

```diff
@@ -57,20 +57,22 @@ See *WordCount Storm* within `flink-storm-examples/pom.xml` for an example how t
 
 Flink provides a Storm compatible API (`org.apache.flink.storm.api`) that offers replacements for the following classes:
 
-- `TopologyBuilder` replaced by `FlinkTopologyBuilder`
 - `StormSubmitter` replaced by `FlinkSubmitter`
 - `NimbusClient` and `Client` replaced by `FlinkClient`
 - `LocalCluster` replaced by `FlinkLocalCluster`
 
-In order to submit a Storm topology to Flink, it is sufficient to replace the used Storm classes with their Flink replacements in the Storm *client code that assembles* the topology.
-The actual runtime code, ie, Spouts and Bolts, can be uses *unmodified*.
-If a topology is executed in a remote cluster, parameters `nimbus.host` and `nimbus.thrift.port` are used as `jobmanger.rpc.address` and `jobmanger.rpc.port`, respectively.
-If a parameter is not specified, the value is taken from `flink-conf.yaml`.
+In order to submit a Storm topology to Flink, it is sufficient to replace the
+used Storm classes with their Flink replacements in the Storm *client code that
+assembles* the topology.  The actual runtime code, ie, Spouts and Bolts, can be
+used *unmodified*.  If a topology is executed in a remote cluster, parameters
+`nimbus.host` and `nimbus.thrift.port` are used as `jobmanger.rpc.address` and
+`jobmanger.rpc.port`, respectively.  If a parameter is not specified, the value
+is taken from `flink-conf.yaml`.
 
```

Comment:
Minor typo "can be uses" -> "can be used". I think fixed line length is the best because sentences can span multiple lines. If it is an issue I can revert it.